book/SUMMARY.md

```diff
@@ -14,6 +14,8 @@
     - [Screen Hook](/plugins/general-abilities/screen-hooks/screen-hooks-general.md)
       - [Android](/plugins/general-abilities/screen-hooks/android/screen-hook-android.md)
       - [iOS](/plugins/general-abilities/screen-hooks/ios/screen-hook-ios.md)
+    - [Video Content Download](/plugins/general-abilities/video-content-download/video-content-download-general.md)
```

Comment:
Please note the updated plugin abilities location in the summary.md file

Reply:
what do you mean, I added there..

Reply:
ok, added to the new place